docs/fields/rich-text.mdx

```diff
@@ -368,3 +369,37 @@ import type {
 } from 'payload/types';
 
 ```
+
+### Selecting fields
```

Comment:
Could be maybe just a reference to the relationship field section?

Reply:
No as I wanted to have a different API for richt-text and relationship/upload